src/db/oracledb/persons-dao.js

```diff
@@ -0,0 +1,32 @@
+import _ from 'lodash';
+
+import { getConnection } from './connection';
+import { contrib } from './contrib/contrib';
+import { serializePerson } from '../../serializers/persons-serializer';
```

Comment:
We have set an alias for the root path, so you should be able to change the path to just `serializers/persons-serializer`